src/main/resources/tasks/clean/clean.js

```diff
@@ -0,0 +1,34 @@
+const libs = {
+    snapshotter: require('/lib/snapshotter'),
+    cluster: require('/lib/xp/cluster'),
+    notifiers: require('/lib/notifiers'),
+    configParser: require('/lib/config-parser'),
+    snapshotJobScheduler: require('/lib/snapshot-job-scheduler'),
+};
+
+exports.run = function (params) {
+    const appConfig = libs.snapshotter.getConfig();
+    const scheduledSnapshotsJobs = libs.configParser.parseSnapshots(appConfig);
+    const text = 'Cleanup';
+
+    try {
+        scheduledSnapshotsJobs.forEach(function (schedule) {
+            cleanScheduledJob(schedule);
+        });
+
+        libs.notifiers.success(text);
+    } catch (e) {
+        libs.notifiers.failed(text, e.message);
+    }
+};
+
+function cleanScheduledJob(schedule) {
+    if (libs.cluster.isMaster()) {
```

Comment:
Ismaster will prevent jobs to run correctly as job is guaranteed to run on one node but not guaranteed to run on master